### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#443] [Esami & Esoneri M-Z]
## Traccia

<img width="605" alt="image" src="https://github.com/user-attachments/assets/cf6fbe57-5da5-4038-b9ca-11b776064645">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/fa6bb4d4-3736-4315-a5de-287c25e3e028)
